extensions/amp-youtube/0.1/validator-amp-youtube.protoascii

```diff
@@ -38,6 +38,8 @@ tags: {  # <amp-youtube>
     mandatory: true
     value_regex: "[^=/?:]+"
   }
+  # <amp-bind>
+  attrs: { name: "[data-videoid]" }
```

Comment:
I'm not sure how to do this one, because we'd end up trying to validate the amp-bind attribute value, which would fail because it's not going to be a valid youtube id.

Probably something like the amp-iframe 'mandatory-oneof' field would work here.